### ISSUE TITLE
[BUG] seq_labeling model训练了二十分钟第一个epoch都没训练完

### ISSUE THREAD

**Issue by wjjwsl** [BrikerMan/Kashgari#98]
你好！我仿照https://eliyar.biz/nlp_chinese_bert_ner/这个示例训练NER的model，代码如下：
<
embedding = BERTEmbedding('bert-base-chinese', sequence_length=30)
model = BLSTMCRFModel(embedding)
model.fit(train_x,
          train_y,
          validate_x,
          validate_y,
          epochs=200,
          batch_size=500)
>
其中，epochs和batch_size都是和这篇文章选的一样的，因为我不知道应该根据什么选……但是我现在已经训练了二十分钟了，连第一个epochs都没有训练完。我用的设备是mac pro，平时训练模型都很快。
print出来的结果如下：
Total params: 108,191,746
Trainable params: 6,884,866
Non-trainable params: 101,306,880
__________________________________________________________________________________________________
Epoch 1/200
14/41 [=========>....................] - ETA: 31:07 - loss: 0.8753 - crf_accuracy: 0.8048

**Comment by BrikerMan** [comment]
正常... BERT 模型就是非常慢...

**Comment by wifeofMikeAngelo** [comment]
那我如果把epoch和batch_size设置得小一点可以么？多少比较合适呢？我看别人的代码里batch_size一般都是32的倍数，epoch是5左右。

**Comment by BrikerMan** [comment]
batch_size 可以根据自己的硬件调整，epoch 只跑几轮也可以，但是可能完全拟合不了数据。通常正式项目的需要迭代很多轮才能达到比较不错的结果。如果在笔记本上跑建议放弃 BERT，用词向量。

**Comment by wjjwsl** [comment]
好的，多谢！

**Comment by BrikerMan** [comment]
可以尝试使用 colab 训练。

**Comment by wjjwsl** [comment]
我试了一下，果然快了很多，多谢！

**Comment by BrikerMan** [comment]
客气，还有，最新版本有些 bug，记得用 0.2.1 版本，或者 0.4.0 之后的版本（还未发布）。
